<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionInterface</class>
 <widget class="QDialog" name="MissionInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>981</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Mission</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>901</width>
       <height>631</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter Mission</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>103</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Date :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ajouter">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="type">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>109</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="datemission">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>147</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>189</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Afficher Mission</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Modifier</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>76</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="type_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>82</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>113</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>162</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>153</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="datemission_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_modif">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>31</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_mission_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Supprimer</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>66</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>94</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Date :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_supp">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>31</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_mission_supp">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="type_val">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="date_val">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="nom_val">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="table_mission">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>350</y>
         <width>811</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_tri_mission">
       <property name="geometry">
        <rect>
         <x>451</x>
         <y>296</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>291</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>291</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>290</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnpdf">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>292</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Imprimer PDF</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Vehicule</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>901</width>
       <height>631</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Ajouter Vehicule</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>103</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>141</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Marque :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ajouter_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="matricule">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>109</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="marque">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Afficher Vehicule</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>361</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Modifier</string>
       </property>
       <widget class="QLineEdit" name="matricule_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>82</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="marque_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_modif_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>31</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_vehicule_modif">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>82</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Marque :</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>331</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>Supprimer</string>
       </property>
       <widget class="QPushButton" name="btn_supp_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>31</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_vehicule_supp">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="matricule_val">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="marque_val">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>108</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Marque :</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="table_vehicule">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>350</y>
         <width>811</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_tri_vehicule">
       <property name="geometry">
        <rect>
         <x>451</x>
         <y>296</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>291</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>291</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche_vehicule">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>290</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>statistique</string>
      </attribute>
      <widget class="QLabel" name="statistiques">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>671</width>
         <height>371</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
